packages/pagebuilder/lib/ContentTypes/Html/configAggregator.js

```diff
@@ -1,12 +1,23 @@
 import { getAdvanced } from '../../utils';
 
 export default node => {
-    const dom = new DOMParser().parseFromString(
-        '<!doctype html><body>' + node.textContent,
-        'text/html'
-    );
+    let html;
+    if (node.dataset.decoded) {
+        html = node.innerHTML;
+        if (process.env.NODE_ENV !== 'production') {
```

Comment:
I think we remove logs in production bundles anyways - shouldn't need to gate with a condition. If I'm wrong we probably have to go clean up our app :D

Reply:
We might, but I just wanted to be explicit. This is a pattern we ought to follow. We may later remove the drop_console in some cases!